### PR TITLE
refactor: clean up GitHub workflow YAML files

### PR DESCRIPTION
**Pull request by jbampton**
- Capitalize first word of `name`
- Remove unneeded full stop
- Remove extra blank line

### What this PR does / why we need it:

Standardize the YAML workflow files.

### Pre-submission checklist:

* [X] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by tokers — APPROVED** [review]

**Comment by Yiyiyimu** [comment]
Hi @jbampton need to merge master to make CI works

**Comment by membphis** [comment]
> Hi @jbampton need to merge master to make CI works

seems that you have done this step, all right? @jbampton

**Review by starsz — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   build:
+    name: Build
```
Comment (spacewander) [inline .github/workflows/build.yml:11]: Does GitHub stop running the action if it doesn't see the name or the name is not capitalized?

I do not see any value added by this PR. And it wastes the CI resource and the review time.

**Review by spacewander — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (jbampton) [reply .github/workflows/build.yml:11]: Having a combination of lowercase and uppercase does not look professional.

Screen Shot 2021-02-22 at 11 58 51 am

**Pull request closed without merging**